docs/source/API/containers/ScatterView.md

```diff
@@ -23,53 +28,53 @@ Kokkos::Experimental::contribute(results, scatter);
 ```c++
 
 template <typename DataType
-        ,int Op
+        ,int Operation
         ,typename ExecSpace
         ,typename Layout
         ,int contribution
         >
 class ScatterView<DataType
```

Comment:
remove all the template arguments blower here i.e. lets just make this:

```c++
template<class DataType, class Layout, class ExecSpace, class Operation, class Contribution>
class ScatterView {
```

And then use Contribution instead of `{ScatterNonDuplicated, ScatterDuplicated}` that just didn't make any sense to write it the way it was. 

Then you need to introduce a Parameters section like in Kokkos::View, before describing the typedefs. 
In the parameters section you describe what the parameters are.

Reply:
@crtrott - apologies for the delayed update.  I'm not sure what the  `class Duplicated` parameter specifies -- would potential values for this parameter be  `ScatterNonDuplicated`, `ScatterDuplicated`?  In any case, I've cleaned the entry up, and tried to get it into a basically correct and current state.